legate.py

```diff
@@ -767,7 +769,22 @@ def driver():
         dest="nsys",
         action="store_true",
         required=False,
-        help="run Legate with nsys",
+        help="run Legate with Nsight",
```

Comment:
I would use "Nsight Systems", to disambiguate from "Nsight Compute".

Reply:
ada8bf41c6b7f5d6778f77b95d6a8771c5e4cde1